cnvlib/commands.py

```diff
@@ -1006,6 +1007,10 @@ def _cmd_scatter(args):
         help="Draw a smoothed local trendline on the scatter plot.")
 P_scatter_aes.add_argument('--y-max', type=float, help="y-axis upper limit.")
 P_scatter_aes.add_argument('--y-min', type=float, help="y-axis lower limit.")
+P_scatter_aes.add_argument('-p', '--plot-dim', nargs=2, type=float,
```

Comment:
How about calling it "figure size" instead of "dim", skipping the short form of the argument (it's not a recurring option in other commands), and setting the metavar for clarity:

    P_scatter_aes.add_argument('--figure-size', nargs=2, metavar=('WIDTH', 'HEIGHT')), type=float,

Reply:
Hi @etal,

Thanks for your code review ! You are right, here is result "help" of commit 85374df3bbecc8aa9805a1ebe3076c461b0b9441 (+ I changed variable name everywhere else):
```python
  --fig-size WIDTH HEIGHT
                        Width and height of the plot in inches. [Default: Pre-defined in Matplotlib 'rcParams'
                        variable (most of the time: '6.4 4.8')]
```

Have a nice day.
Felix.